### PR TITLE
Layer5 ng_#833

### PR DESCRIPTION
**Pull request by jomefavourite**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@mdx-js/mdx": "^1.6.11",
     "@mdx-js/react": "^1.6.11",
     "babel-plugin-styled-components": "^1.10.7",
-    "gatsby": "^2.19.32",
+    "gatsby": "^2.24.65",
```
Comment (Tanuj22) [inline package.json:27]: revert this please

File: src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (Tanuj22) [inline src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx:1]: The format of this file should be like the existing news article present. Please look at them.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
@jomefavourite The format of the file should be like the existing news article present not like ones in the current site otherwise it won't show up.

**Comment by jomefavourite** [comment]
Okay, but I'm having a hard time , since im not conversant with React. If you could help out...

**Comment by Tanuj22** [comment]
@jomefavourite have a look at this file - https://raw.githubusercontent.com/layer5io/layer5/layer5-ng/src/collections/news/0000-00-00-news-template/index.mdx .  You have to replace the title, erul, etc. accordingly. The the content would be between the opening and closing `<NewsWrapper>` tags.

**Comment by leecalcote** [comment]
@jomefavourite thanks for plugging away here. It's inspiring to see you breaking into Reactjs. 💪

**Comment by jomefavourite** [comment]
@leecalcote Thanks

**Comment by jomefavourite** [comment]
> @jomefavourite have a look at this file - https://raw.githubusercontent.com/layer5io/layer5/layer5-ng/src/collections/news/0000-00-00-news-template/index.mdx .  You have to replace the title, erul, etc. accordingly. The the content would be between the opening and closing `<NewsWrapper>` tags. 

Thanks for making it clear

**Comment by Tanuj22** [comment]
done in #887

**Pull request closed without merging**